### PR TITLE
Basic writeup of security discussions

### PR DESCRIPTION
**Pull request by curry684**

### CODE REVIEW

**Review by jfhovinne — APPROVED** [review]
Thanks !

File: achievements/security.md [hunk 1]
```diff
@@ -1 +1,14 @@
+# Symfony Security
+
+On the subject of security we had some great discussions and research about the state of the
+Symfony Security component as it is now. I has not received much love over the past years and
```
Comment (OskarStark) [inline achievements/security.md:4]: I -> It

Comment (curry684) [reply achievements/security.md:4]: I haz also not received much love over the past year 😢

Comment (jfhovinne) [reply achievements/security.md:4]: Thanks for the review @OskarStark , was fixed by 95a5db1

**Pull request merged**